vl-proza-message.src.js

```diff
@@ -79,15 +79,13 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
     }
 
     static _getMessage(domain, code) {
-        if (VlProzaMessagePreloader.isPreloaded(domain)) {
-            return VlProzaMessagePreloader.getMessage(domain, code);
-        } else {
+        return VlProzaMessagePreloader.getMessage(domain, code).catch(() => {
```

Comment:
Er wordt geen rekening meer gehouden met de preload?

Reply:
Jawel, in de VlProzaMessagePreloader.getMessage zal ook worden gekeken of het domein preloaded is of niet. Zo nee, wordt er een rejected promise teruggegeven.